projects/igniteui-angular/src/lib/combo/combo.component.ts

```diff
@@ -1249,9 +1273,7 @@ export class IgxComboComponent extends DisplayDensityBase implements IgxComboBas
      */
     public ngOnInit() {
         this.ngControl = this.injector.get(NgControl, null);
-        this._positionCallback = () => this.dropdown.updateScrollPosition();
-        this.overlaySettings.positionStrategy = new ComboConnectedPositionStrategy(this._positionCallback);
-        this.overlaySettings.positionStrategy.settings.target = this.elementRef.nativeElement;
+        this.overlaySettings.positionStrategy.settings.target = this.element;
```

Comment:
At this point you are overriding the position target that may be passed by the user. Does that the user can't set a different target?
And this will only run once and won't adjust runtime changes if that's the goal.